### PR TITLE
Homework

### PR DESCRIPTION
**Pull request by ivombi**
This PR contains the following changes

1. I created questions
2. Assign those questions to students

### CODE REVIEW

**Review by BihBerniceF — CHANGES_REQUESTED** [review]
Please can you add the line 'practice session'?

**Comment by ivombi** [comment]
> Please can you add the line 'practice session'?

@BihBerniceF I think removing make sense.

**Review by BihBerniceF — APPROVED** [review]
Good to merge

**Review by ghost — COMMENTED** [review]
nice one mate. crack on and merge.

File: README.md [hunk 1]
```diff
@@ -1,2 +1,28 @@
...
+For your assignment, answer the following questions
+
+1. What is git and github? (Bernice)
+
+
+2. What is a repository?(Lovis)
+
+
+3. What is a branch? (Ted)
+
```
Comment (tyanko55) [inline README.md:12]: it is a source which allows you merge feature ideas  in a contained area of repository .

**Pull request merged**